cloud/pkg/edgecontroller/manager/const_test.go

```diff
@@ -0,0 +1,26 @@
+package manager
+
+const mockKubeConfigContent = `
```

Comment:
we need this const?

Reply:
In package manager, unittest need to create an k8s client, so we need a temp kubeconfig file. When I put this string into the test method, command `make verify` will run error, and warn this string is used in many files. But i didn‘t find a file which is suitable to add this string only for unittest, so i put it here. Otherwise i think we should check the unittest code more strictly, because the old unittest in package manager will skip when create kubeclient failed, and not really cover all mehod logic when github  or travis ci runs.